backend/app/api/routes/api_keys.py

```
@@ -1,116 +1,81 @@
+@router.post(
+    "/",
+    response_model=APIResponse[APIKeyCreateResponse],
+    status_code=201,
+    dependencies=[Depends(require_permission(Permission.SUPERUSER))],
+)
+def create_api_key_route(
     project_id: int,
     user_id: int,
-    session: Session = Depends(get_db),
-    current_user: User = Depends(get_current_active_superuser),
+    current_user: AuthContextDep,
```

Ruff (0.13.3)

Unused function argument: `current_user`

(ARG001)